ledger/.gitignore

```diff
@@ -0,0 +1 @@
+catchpoints
```

Comment:
@jannotti Sanity checking:  Is the `catchpoints` gitignore addition a by-design change?

It's unclear to me if the related Slack discussion reached an outcome, so I'm leaving the double check question.

Reply:
Yeah, nobody seemed to explicitly want it left here, and I'm tired of my seeing my repo marked dirty because of a random generated file.

Reply:
@jannotti Alrighty - Resolving.